libmamba/src/api/configuration.cpp

```diff
@@ -1423,6 +1423,14 @@ namespace mamba
                    .set_env_var_names()
                    .set_post_merge_hook(detail::not_supported_option_hook));
 
+        insert(Configurable("experimental_matchspec_parsing", &m_context.experimental_matchspec_parsing)
+                   .group("Basic")
+                   .description(  //
+                       "Enable experimental parsing and matching of MatchSpecs using Mamba implementation.\n"
```

Comment:
Nitpick to disambiguate given than `MatchSpec` are already use externally (e.g. by `conda-libmamba-solver`).
```suggestion
                       "Enable internal parsing and matching of MatchSpecs using Mamba's experimental implementation rather than Libsolv's.\n"
```